types/react-native-video/index.d.ts

```diff
@@ -0,0 +1,77 @@
+// Type definitions for react-native-swiper 1.0
+// Project: https://github.com/react-native-community/react-native-video
+// Definitions by: HuHuanming <https://github.com/huhuanming>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+import * as React from 'react';
+import {
+    ViewProperties
+} from 'react-native';
+
+export interface VideoProperties extends ViewProperties {
+    /* Native only */
+    src?: any;
```

Comment:
This comment is going to show up as the documentation for the `src` prop, which doesn’t seem the intention, no?

How about using the docs from [the usage example](https://github.com/react-native-community/react-native-video#usage)?